packages/pigeon/lib/kotlin_generator.dart

```diff
@@ -628,7 +629,7 @@ void generateKotlin(KotlinOptions options, Root root, StringSink sink) {
         if (klass.fields.last != element) {
           indent.addln(',');
         } else {
-          indent.addln('');
+          indent.add('');
```

Comment:
Same question here.

Reply:
this change reduce the blank line in data class.
```kotlin
data class Book (
  val title: String? = null,
  val author: String? = null

) {
```